src/Middleware/SpaServices/src/Prerendering/PrerenderTagHelper.cs

```diff
@@ -101,8 +101,10 @@ public override async Task ProcessAsync(TagHelperContext context, TagHelperOutpu
 
             if (!string.IsNullOrEmpty(result.RedirectUrl))
             {
+                var permanentRedirect = result?.StatusCode == 301;
```

Comment:
No need for the `?` in this line. We already know from L102 that `result` is nonnull.

Reply:
That's true, thanks for looking at it! I updated the PR.